src/constants/Footer.js

```diff
@@ -3,5 +3,23 @@ const FOOTER = {
   SUBSCRIBE: "Subscribe",
   CONTACT_US: "Contact us",
   PHONE_NUMBER: "+(380) 98 56 234",
+  PASTE_YOUR_EMAIL: "Paste your email",
+  SEARCH_EVENT: "Search event",
+  HOME: "Home",
+  CREATE_EVENT: "Create event",
+  TERMS: "Terms",
+  ABOUT: "About",
+  PATH_HOME: "/home",
```

Comment:
Can we group some of these constants into few categories
For examples: All PATH_* into PATHS, LINK_* into LINKS and so one

Reply:
Done